datastore/tests/unit/datastore_test.py

```diff
@@ -1,5 +1,23 @@
-import gcloud.aio.datastore.datastore as datastore  # pylint: disable=unused-import
+import gcloud.aio.datastore.datastore as datastore
+from gcloud.aio.datastore.constants import TypeName
+from gcloud.aio.datastore.utils import TYPES
 
 
-def test_importable():
-    assert True
+class TestDatastore:
```

Comment:
Do we need to make a class for this? With `pytest` you can just use a function to represent the test you want to run.

Reply:
That's true but that's not what we do in the other test files, so I thought I'd add a class here too. There may be more tests in this file in the future.

Reply:
@KevinGDialpad yeah, I'd been meaning to change that -- we prefer test functions over test classes, unless there's some reason why the test class would be necessary for this case (which I don't think is the case here). Please nuke the class!

Reply:
I see. I'll remove this one then!